### ISSUE TITLE
VM for 

### ISSUE THREAD

**Issue by GoogleCodeExporter** [kuza55/opendlp#19]
```
When do you think a vm for the newer build 0.3.1 or higher of OpenDLP will be 
released?  Or will there be one?
```

Original issue reported on code.google.com by `dvanme...@kumc.edu` on 20 May 2011 at 3:39

**Comment by GoogleCodeExporter** [comment]
```
I will be releasing a new one after OpenDLP 0.3.2 is released.  Hopefully this 
will happen in a few weeks.  I may also release both VirtualBox and VMware 
images to make it easier for people.
```

Original comment by `andrew.O...@gmail.com` on 20 May 2011 at 3:47

**Comment by GoogleCodeExporter** [comment]
```
I don`t understand how "ubuntu-based" works. I have to install a VM image of 
ubuntu first? and after that how do I import this images of opendlp? could 
someone help me pls with this? I am new in virtualbox :( I ussualy don`t work 
with virtual machines
```

Original comment by `AndreiMa...@gmail.com` on 20 Jun 2011 at 12:00

**Comment by GoogleCodeExporter** [comment]
```
I just released a new VirtualBox VM based on Ubuntu 11.04 and OpenDLP 0.4.1. 
You can download it here: https://code.google.com/p/opendlp/downloads/list
```

Original comment by `andrew.O...@gmail.com` on 15 Aug 2011 at 6:03
- Changed state: **Fixed**
